draft-ietf-rtcweb-jsep.xml

```diff
@@ -1768,11 +1768,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             the media bandwidth, RTCP fraction allocated to the senders
             and setting maximum media bit-rate boundaries.</t>
 
+            <t>
+            <xref target="RFC4566"></xref>, Section 5.13, JSEP 
+            implementations MUST ignore unknown attribute (a=) lines.</t>
+
```

Comment:
I see why you moved this in, but I think it makes more sense out of the list, because it isn't so much a thing you have to implement but a general rule. But I can go either way. @juberti ?